client/package-lock.json

```diff
@@ -935,7 +935,6 @@
       "version": "7.9.6",
```

Comment:
There are a lot of changes in this file! Can you please remove it from the pull request, and I will generate it locally. Thanks!

Reply:
client/package.lock.json file removed.